Client/Impl/Worker/JobWorker.cs

```diff
@@ -73,18 +72,18 @@ internal void Open()
                     async () =>
                         await Poll(cancellationToken)
                             .ContinueWith(
-                                t => Logger.ErrorException("Job polling failed.", t.Exception),
+                                t => logger?.LogError(t.Exception, "Job polling failed."),
```

Comment:
Is there then a NullLogger if we used the NullFactory?

Reply:
there is but i pulled the NullLoggerFactory due to the IDisposable warnings.